ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/committer/transaction/TransactionCommitter.scala

```diff
@@ -69,12 +69,14 @@ private[kvutils] class TransactionCommitter(
   private val committerModelConformanceValidator =
     new CommitterModelConformanceValidator(engine, metrics)
 
+  /** The order of the steps matters
+    */
   override protected val steps: Steps[DamlTransactionEntrySummary] = Iterable(
+    "set_time_bounds" -> TimeBoundBindingStep.setTimeBoundsInContextStep(defaultConfig),
+    "validate_ledger_time" -> ledgerTimeValidator.createValidationStep(rejections),
     "authorize_submitter" -> authorizeSubmitters,
```

Comment:
Changed the order to ensure the out of time bounds entry is set before any rejection can stop execution